components/Main/StyledExercises.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+
+export const StyledExercises = styled.ul`
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  margin: 0;
+  li {
+    margin: 20px;
+    padding: 10px;
+    margin-bottom: 0;
+    border-radius: 10px;
+    list-style: none;
+    background-color: white;
+    color: #354f52;
+    text-align: center;
+    box-shadow: 1px 1px 3px 3px #cad2c5;
+  }
+  h2 {
+    font-family: Arial, Helvetica, sans-serif;
+    display: flex;
+    justify-content: center;
+    padding: 5px;
+    margin-top: 20px;
+    margin-bottom: 0;
```

Comment:
not sure if margin-bottom needs to be set to 0 (refer to GlobalStyles.js) but you could alternatively use one line again if you wish